.github/workflows/ci-test.yaml

```diff
@@ -5,14 +5,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
-        with:
-          go-version: 1.15
```

Comment:
This was bypassing the go version we had in our build scripts since it was trying to run local unit tests instead of dockerized. Local is really just for dev